core/src/main/res/values/strings.xml

```diff
@@ -371,33 +371,33 @@
-    <string name="pref_hardware_previous_button_title">Previous Button</string>
-    <string name="pref_hardware_previous_button_summary">Customize the previous button behavior</string>
-    <string name="button_action_fast_forward">Fast Forward</string>
-    <string name="button_action_rewind">Rewind</string>
-    <string name="button_action_skip_episode">Skip Episode</string>
-    <string name="button_action_restart_episode">Restart Episode</string>
+    <string name="pref_unpauseOnBluetoothReconnect_sum">Resume playback when Bluetooth reconnects</string>
+    <string name="pref_hardware_forward_button_title">\"Forward\" Button</string>
+    <string name="pref_hardware_forward_button_summary">Customize the \"Forward\" button behavior</string>
+    <string name="pref_hardware_previous_button_title">\"Previous\" Button</string>
+    <string name="pref_hardware_previous_button_summary">Customize "\Previous\" button behavior</string>
+    <string name="button_action_fast_forward">Fast Forward…</string>
+    <string name="button_action_rewind">Rewind…</string>
+    <string name="button_action_skip_episode">Skip Episode…</string>
+    <string name="button_action_restart_episode">Restart Episode…</string>
     <string name="pref_followQueue_sum">Jump to next queue item when playback completes</string>
-    <string name="pref_auto_delete_sum">Delete episode when playback completes</string>
-    <string name="pref_auto_delete_title">Auto Delete</string>
-    <string name="pref_smart_mark_as_played_sum">Mark episodes as played even if less than a certain amount of seconds of playing time is still left</string>
-    <string name="pref_smart_mark_as_played_title">Smart Mark as Played</string>
-    <string name="pref_skip_keeps_episodes_sum">Keep episodes when they are skipped</string>
+    <string name="pref_auto_delete_sum">Delete played episodes</string>
+    <string name="pref_auto_delete_title">Auto-delete</string>
+    <string name="pref_smart_mark_as_played_title">Smart-mark as Played</string>
+    <string name="pref_smart_mark_as_played_sum">Mark episodes as played even with less than a set amount of seconds of playtime left</string>
     <string name="pref_skip_keeps_episodes_title">Keep Skipped Episodes</string>
-    <string name="pref_favorite_keeps_episodes_sum">Keep episodes when they are marked Favorite</string>
-    <string name="pref_favorite_keeps_episodes_title">Keep Favorite Episodes</string>
+    <string name="pref_skip_keeps_episodes_sum">Keep episodes that are skipped</string>
+    <string name="pref_favorite_keeps_episodes_title">Keep Favourited Episodes</string>
+    <string name="pref_favorite_keeps_episodes_sum">Keep episodes marked as favourites</string>
     <string name="playback_pref">Playback</string>
-    <string name="playback_pref_sum">Headphone controls, Skip intervals, Queue</string>
+    <string name="playback_pref_sum">Headphone controls, skip intervals, queue</string>
```

Comment:
I am probably wrong here, but these aren't proper nouns.
Do you mean they could be mistaken? "Headphone controls" is clear enough, "skip intervals" I get, (which could be "skip-intervals", but then it would be "enqueue".